src/com/dynatrace/loadrunner/LoadRunnerIntegrationTests.java

```diff
@@ -0,0 +1,151 @@
+package com.dynatrace.loadrunner;
+
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardCopyOption;
+import java.util.LinkedList;
+import java.util.List;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+import com.dynatrace.loadrunner.logic.LoadRunnerConverter;
+import com.dynatrace.loadrunner.logic.ScriptFile;
+
+public class LoadRunnerIntegrationTests {
+
+	@Rule
+	public TemporaryFolder tempFolderRule = new TemporaryFolder();
+
+	private Path cHeaderFilePath;
```

Comment:
seems like a [Parameterized](https://junit.org/junit4/javadoc/4.12/org/junit/runners/Parameterized.html) JUnit test; use it instead